merchants/verkkokauppa/helpers.py

```diff
@@ -59,6 +59,26 @@ def get_validated_phone_number(phone_number: str) -> str:
     return ""
 
 
+def get_meta_label(key: str, reservation: Reservation) -> str:
+    labels = {
+        "reservationPeriod": {
+            "fi": "Varausaika",
+            "en": "Booking time",
+            "sv": "Bokningstiden",
+        },
+        "reservationNumber": {
+            "fi": "Varausnumero",
+            "en": "Booking number",
+            "sv": "Bokningsnummer",
+        },
+    }
+    if key not in labels:
+        raise Exception(f"Invalid meta label key '{key}")
```

Comment:
Consider creating a own Exception type for this

Reply:
Done! I also added more tests 👍